travis_build.sh

```diff
@@ -30,10 +30,14 @@ git_clean() {
 
 build() {
     ./build.sh
-    CONFIGURE=$(readlink -f configure)
+    CONFIGURE=$(realpath configure)
     mkdir -p build
     pushd build
     configure "$CONFIGURE"
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+      # create workaround setup.cfg for installing python package in mac os
+      echo $'[install]\nprefix='>src/nrnpython/setup.cfg
```

Comment:
CMAKE: Workaround for an error during python installation in mac
error: must supply either home or prefix/exec-prefix -- not both